appveyor.yml

```diff
@@ -1,9 +1,32 @@
 version: 1.0.{build}
+
+environment:
+  NSIS_ROOT: C:\nsis
+
+install:
+  - appveyor DownloadFile "http://downloads.sourceforge.net/project/nsis/NSIS 2/2.46/nsis-2.46-setup.exe" -FileName nsissetup.exe
+  - nsissetup /S /D=%NSIS_ROOT%
+
 before_build:
 - make dependencies
+
 build:
   project: OpenRA.sln
   verbosity: minimal
+
 test_script:
-- nunit-console-x86.exe OpenRA.Test.dll
-deploy: off
\ No newline at end of file
+  - nunit-console-x86.exe OpenRA.Test.dll
+
+after_build:
+  - cd packaging/windows
+  - '%NSIS_ROOT%\makensis /DSRCDIR="." /DDEPSDIR="./thirdparty/windows" OpenRA.nsi'

```

Comment:
if you cd into `packing/windows`, the SRCDIR and DEPSDIR paths need to be adjusted. Or you don't move down into that directory and give the full path to OpenRA.nsi instead.